src/compass/utils/h5_helpers.py

```diff
@@ -370,10 +370,10 @@ def identity_to_h5group(dst_group, burst):
             'Azimuth stop time of product'),
         Meta('list_of_frequencies', ['A'],
              'List of frequency layers available in the product'),  # T)C
-        Meta('is_geocoded', True, 'Flag to indicate radar geometry or geocoded product'),
-        Meta('is_urgent_observation', False,
+        Meta('is_geocoded', 'True', 'Flag to indicate radar geometry or geocoded product'),
+        Meta('is_urgent_observation', 'False',
              'List of booleans indicating if datatakes are nominal or urgent'),
```

Comment:
Is this a "List of booleans"?
Would we need to change this to `string of "True" or "False" if we're making it a string?

Reply:
I suspect that the "list" comes from NISAR but it is not really needed for CSLC-S1 as each product will contain only a single burst.